indexer/e2e_tests/bridge_transactions_e2e_test.go

```diff
@@ -0,0 +1,170 @@
+package e2e_tests
```

Comment:
wow these tests are great and comprehensive great job!

Reply:
thanks!